### PR TITLE
fix: scaffold commands file

### PR DESCRIPTION
**Pull request by estrada9166**
- Closes  [UNIFY-1192](https://cypress-io.atlassian.net/browse/UNIFY-1192?filter=10053)

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/launchpad/cypress/e2e/project-setup.cy.ts [hunk 1]
```diff
@@ -737,6 +743,7 @@ describe('Launchpad: Setup Project', () => {
           cy.contains('cypress.config.ts')
           cy.containsPath('cypress/component/index.html')
           cy.containsPath(`cypress/support/component.ts`)
+          cy.containsPath(`cypress/support/commands.ts`)
```
Comment (lmiller1990) [inline packages/launchpad/cypress/e2e/project-setup.cy.ts:746]: Can we take this chance to add some tests to actually verify the file is **created** on the filesystem? These tests just verify we show the file we will create in the UI, not that they are actually written.

We have a pattern for this, see this function: https://github.com/cypress-io/cypress/blob/10.0-release/packages/launchpad/cypress/e2e/scaffold-component-testing.cy.ts#L11-L19 . This one actually verifies the content, too (might be worth doing something similar here?)

**Review by lmiller1990 — CHANGES_REQUESTED** [review]
cool, would be good to make the tests a little more thorough if possible, what do you think?

**Review by ZachJW34 — DISMISSED** [review]
Works great! Might be worth adding a system test to verify what we scaffold works when all linked up, like adding a custom command to `commands.js` is usable in a test, but doesn't have to be in this PR

File: packages/data-context/src/actions/WizardActions.ts [hunk 2]
```diff
@@ -375,6 +378,61 @@ export class WizardActions {
...
+      import './commands'
+  
+      // Alternatively you can use CommonJS syntax:
+      // require('./commands')
+    `
+  }
+
+  private commandsFileBody (language: CodeLanguageEnum) {
+    return dedent`
+      // ***********************************************
```
Comment (emilyrohrbough) [inline packages/data-context/src/actions/WizardActions.ts:409]: If the language is typescript, Should this file include the example types for these custom commands as well?

**Review by emilyrohrbough — APPROVED** [review]

**Review by lmiller1990 — APPROVED** [review]

**Pull request merged**